src/web/features.rs

```diff
@@ -259,7 +260,7 @@ mod tests {
                 .create()?;
 
             let resp = env.frontend().get("/crate/foo/latest/features").send()?;
-            assert!(resp.url().as_str().ends_with("/crate/foo/0.2.0/features"));
+            assert!(resp.status().is_success());
```

Comment:
That's a bit odd since the goal is to test that this serves immediately (200, ~= is_success) rather than redirects (302). The version you propose would pass if we redirected to self (or to any other path that ends with the requested URL)

Reply:
@jsha `reqwest.get()` automatically follows redirects. If you want to test that there's no intermediate redirects, you should call `override_frontend`: https://github.com/rust-lang/docs.rs/blob/495e56c92a827888f4ae78a05e35f7e639f39079/src/web/rustdoc.rs#L1032-L1037

Once you set it not to follow redirects, I still think it makes sure to test the path directly to make sure it's going to the right page.